### DISCUSSION TITLE
FR: add support for colored emoji on device names

### DISCUSSION THREAD

**Discussion by alefranz** [File-New-Project/EarTrumpet#633] [Ideas]
Having an emoji in the device name is an easy way to immediately spot the right device.
On the windows one they displayed in color, which makes it really nice.
![image](https://user-images.githubusercontent.com/348793/80958752-650fd480-8dfd-11ea-980f-cda789041da7.png)
It would be nice to make it colored even in EarTrumpet :)
![image](https://user-images.githubusercontent.com/348793/80958734-5c1f0300-8dfd-11ea-9e89-900aabbef533.png)

**Comment by riverar** [comment]
Hah, I actually already added this to our internal backlog. I like it.

**Comment by liuxilu** [comment]
I'd like to have a note, please fully implemente [Emoji and Text Presentation Sequences](https://www.unicode.org/reports/tr51/#Emoji_Variation_Sequences), to let users control the color behavior.
